app/clients/letter/dvla.py

```diff
@@ -80,5 +103,99 @@ def authenticate(self):
 
         return response.json()["id-token"]
 
-    def send_letter(self):
-        pass
+    def _get_auth_headers(self):
+        return {
+            "Accept": "application/json",
+            "Authorization": self.jwt_token,
+            "X-API-Key": self.dvla_api_key.get(),
+        }
+
+    def send_letter(
+        self,
+        notification_id: str,
+        address: list[str],
+        postage: str,
+        service_id: str,
+        organisation_id: str,
+        pdf_file: bytes,
+    ):
```

Comment:
I wonder if we should prefer to use keyword-only arguments here given we have quite a few things to pass in and calling it without keywords is going to make it hard to verify on-sight that the args in the right order. One way to do this:

```suggestion
    def send_letter(
        self,
        /,
        notification_id: str,
        address: list[str],
        postage: str,
        service_id: str,
        organisation_id: str,
        pdf_file: bytes,
    ):
```

What do you think?

Reply:
Agree this could help avoid confusion, particularly between the service_id and organisation_id, so I've made this change 👍